source/guides/basics/mechanisms.md

```diff
@@ -1,49 +1,150 @@
-Mechanisms And Properties (PLACEHOLDER ONLY)
+Mechanisms And Properties
 -------------------------
 
 ```eval_rst
 .. contents:: Table of Contents
     :local:
 ```
 
-### Placeholder
+### What's a Mechanism?
 
-Until this page is written, you can view the [old tutorial video here](https://one.denizenscript.com/denizen/vids/Properties%20and%20Mechanisms).
+If you've made it this far, you've learned about [definitions](/guides/basics/definitions) and [flags](/guides/basics/flags), the two most common ways to read and write stored data for use in your scripts. If you're reading this guide for the first time, though, you're probably here to *change* data, not just read and write it. **Mechanisms** are how Denizen parses and manages data related to objects. Here are a few examples of their use in script containers:
+
+```dscript_green
+monkey_wrench:
+    type: item
+    display name: Monkey Wrench
+    material: iron_hoe
+    lore:
+    - A monkey wrench.
+    - Useful for tinkering with mcmonkey4eva.
+    mechanisms:
+        unbreakable: true
+```
+
+This item script, using the `mechanisms` key and the `unbreakable` mechanism with a value of `true`, makes the item unbreakable.
+
+```dscript_green
+undead_golden_swordsman:
+    type: entity
+    entity_type: zombie
+    item_in_hand: golden_sword
+    item_in_offhand: golden_sword
+    health_data: 80/80
+```
+
+This entity script creates a zombie and uses the `item_in_hand` and `item_in_offhand` mechanisms to put golden swords in both hands. It also spawns with 80 health, and has 80 health at maximum, using the `health_data` mechanism.
 
 ### The Adjust Command
 
-The `adjust` command is primarily used to adjust entities that are already spawned or otherwise have a unique identifier.
+Mechanisms can be used in more than just script containers! If you need to alter the value of a mechanism, the `adjust` command is the way to go. The `adjust` command is primarily used to adjust entities that are already spawned or otherwise have a unique identifier.
+
+For example, if we want to write a script that increases the max health of the swordsman we made above, we could do it like this:
+
+```dscript_green
+undead_golden_swordsman_growth:
+    type: world
+    events:
+        on entity killed by undead_golden_swordsman:
+        - define current_damage <context.damager.attribute_value[generic_attack_damage]>
+        - adjust <context.damager> attributes:generic_attack_damage/<[current_damage].add[1]>
```

Comment:
The attributes mech is probably not ideal for an initial example, as it uses a rather odd format (which will probably get changed in the future as well).
Also, initial examples should be really easy to see and test. Maybe something more like the `color` mechanism when a player clicks a sheep with a certain tool.